win32/src/threadpool_win32.c

```diff
@@ -86,20 +90,16 @@ static void internal_close(THREADPOOL_HANDLE threadpool)
         {
             break;
         }
-
         (void)WaitOnAddress(&threadpool->pending_api_calls, &current_pending_api_calls, sizeof(current_pending_api_calls), INFINITE);
     } while (1);
 
-    /* Codes_SRS_THREADPOOL_WIN32_01_030: [ threadpool_close shall wait for any executing callbacks by calling CloseThreadpoolCleanupGroupMembers, passing FALSE as fCancelPendingCallbacks. ]*/
-    CloseThreadpoolCleanupGroupMembers(threadpool->tp_cleanup_group, FALSE, NULL);

```

Comment:
>CloseThreadpoolCleanupGroupMembers [](http://example.com/codeflow?start=4&length=34)

Sorry I don't know the background on this change, but why remove this? Calling `CloseThreadpoolCleanupGroupMembers` with `fCancelPendingCallbacks` `FALSE` means we wait for currently executing threads to complete...but if we remove this then doesn't that mean there can be threadpool threads running after the threadpool is closed?

Reply:
I sent you the reasoning out-of-band, let me know if you have any questions or comment.